app/controllers/api/services_controller.rb

```diff
@@ -128,6 +128,7 @@ def can_edit_endpoints?
 
   # This will be the default 'settings' when apiap is live
   def settings_page
+    # TODO: remove all :settings associated pages
```

Comment:
It helps for later to search if all the `TODO` related to this issue start with `# TODO: THREESCALE-3759` as you have done in other PRs 😄